src/http/get-clear_questions/.arc-config

```diff
@@ -0,0 +1,5 @@
+@aws
+runtime nodejs12.x
+# memory 1152
+# timeout 30
+# concurrency 1
```

Comment:
I don't necessary have a strong opinion about this, primarily because I don't know how many people would use this function. If we load test this at 50 requests per sec, we can see how it would perform.

Further reading:
https://arc.codes/reference/arc-config/concurrency
https://docs.aws.amazon.com/lambda/latest/dg/invocation-scaling.html